### PR TITLE
change bert pretraining to stop on total number of iterations

### PR DESCRIPTION
**Pull request by yzhang123**
Signed-off-by: Yang Zhang <yangzhang@nvidia.com>

### CODE REVIEW

File: examples/nlp/bert_pretraining.py [hunk 1]
```diff
@@ -60,6 +60,7 @@
 """
 import argparse
 import os
+import math
```
Comment (chiphuyen) [inline examples/nlp/bert_pretraining.py]: please see style guide for imports
https://www.python.org/dev/peps/pep-0008/#imports

File: examples/nlp/bert_pretraining.py [hunk 2]
```diff
@@ -113,7 +114,8 @@
 parser.add_argument("--only_mlm_loss", action="store_true",
                     default=False, help="use only masked language model loss")
 parser.add_argument("--total_iterations_per_gpu", default=-1,
```
Comment (chiphuyen) [inline examples/nlp/bert_pretraining.py]: do you have use cases for this?

Comment (okuchaiev) [reply examples/nlp/bert_pretraining.py]: yes, but "total_iterations_per_gpu" is a very confusing name. Why just not name this "max_steps"?

File: examples/nlp/bert_pretraining.py (hunk 1 above)
Comment (yzhang123) [reply examples/nlp/bert_pretraining.py]: i dont understand whats the problem?

**Pull request merged**